contract_manager/src/contracts/evm.ts

```diff
@@ -598,7 +598,7 @@ export class EvmPriceFeedContract extends PriceFeedContract {
       pythContract.methods.updatePriceFeeds(priceFeedUpdateData);
     const gasEstiamte = await transactionObject.estimateGas({
       from: address,
-      gas: 100000000,
+      gas: 15000000,
```

Comment:
the previous number was too high for Hedera. @m30m can we get away from this parameter entirely?

Reply:
I tried to include paramters only if necessary. Don't clearly remember why we needed this one but there is a high chance that removing it will break something 😓